### PR TITLE
Rename “cheat sheet” page

### PR DESCRIPTION
**Pull request by aakashdeshamne**
Change the title of cheatsheet.md file “kubectl cheat sheet” to “kubectl Quick Guide”

### CODE REVIEW

**Comment by Gauravpadam** [comment]
Targets #43433

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: kubectl Cheat Sheet
+title: kubectl Quick Guide
```
Comment (Gauravpadam) [inline content/en/docs/reference/kubectl/cheatsheet.md:2]: This only fixes the title of the page, I still have to make a call to kubectl/cheatsheet endpoint though
image

**Review by Gauravpadam — COMMENTED** [review]
Thanks for the PR @aakashdeshamne,

Here's what I'd suggest,
Try renaming the file to something like `quickguide.md`, That should fix the endpoint as well

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (aakashdeshamne) [reply content/en/docs/reference/kubectl/cheatsheet.md:2]: thanks for suggestion Gaurav

Comment (aakashdeshamne) [reply content/en/docs/reference/kubectl/cheatsheet.md:2]: can you please tell me that the issue will solve by just changing filename

Comment (Gauravpadam) [reply content/en/docs/reference/kubectl/cheatsheet.md:2]: Yes, It should. Push and I'll check again for you

Comment (aakashdeshamne) [reply content/en/docs/reference/kubectl/cheatsheet.md:2]: can you please check it again

**Review by Gauravpadam — COMMENTED** [review]
Endpoint is fixed,
image

Thanks, LGTM

**Comment by aakashdeshamne** [comment]
> Endpoint is fixed, image
> 
> Thanks, LGTM
What does I have to do for this PR getting merge

**Comment by aakashdeshamne** [comment]
@Gauravpadam  Can you please tell me what I can do extra for this pr getting merge

**Comment by Gauravpadam** [comment]
Please be patient, It usually takes a few days for approvers to approve a PR

**Comment by kbhawkey** [comment]
Related to https://github.com/kubernetes/website/pull/43665.

**Comment by kbhawkey** [comment]
Hello @aakashdeshamne . Thanks for contributing.
Pull request https://github.com/kubernetes/website/pull/43665 has addressed the issue. I plan to close this pull request.
/close

**Pull request closed without merging**